.github/workflows/release.yml

```diff
@@ -0,0 +1,197 @@
+name: Release Pipeline
+
+on:
+  push:
+    branches:
+      # Regular release channels
+      - master
+      - next
+      - beta
+      - alpha
+      # Support, hotfix branches like: 1.0.x or 1.x
+      - "+([0-9])?(.{+([0-9]),x}).x"
```

Comment:
is this regex correct?
I'm not sure but I tested on validator and I think it should be: `([0-9])+(.{+([0-9]),x})?.x`

Reply:
Nice catch! That's actually a pattern for `micromatch`, so I patched the regex used in the workflow files to a valid reg-exp.